root/utility/sanitizedEditor.js

```diff
@@ -7,7 +7,20 @@
  * later version: http://www.gnu.org/licenses/gpl-2.0.txt
  */
 
-export default function sanitizedEditor(editor: EditorT): SanitizedEditorT {
+export default function sanitizedEditor(editor: EditorT | SanitizedEditorT | null): SanitizedEditorT {
+  if (!editor) {
+    editor = {
```

Comment:
It does feel a bit weird to add this here, since this function is for removing private data. Can we just avoid calling `sanitizedEditor` to begin with by modifying the check at https://github.com/metabrainz/musicbrainz-server/blob/6643637/root/static/scripts/common/components/Annotation.js#L126 to `if (props.annotation && props.annotation.editor)` with a comment explaining when the editor is null?

Reply:
Agreed, done in https://github.com/metabrainz/musicbrainz-server/pull/792/commits/da3428846427526719bf706eaa3af9c2bc6afefe, I also set `editor` property in `AnnotationT` type to be potentially `null`.